radiation/get_data.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ -z "$IS_DOCKER" ] ; then
```

Comment:
you still need the regression data if not using docker, how does the code get it in that case?